osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/infra/messaging/CoreLogItemObjectMessageCreator.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.core.infra.messaging;
+
+import javax.jms.JMSException;
+import javax.jms.ObjectMessage;
+import javax.jms.Session;
+import org.opensmartgridplatform.shared.infra.jms.Constants;
+
+public class CoreLogItemObjectMessageCreator {
```

Comment:
An implementation more similar to `LoggingObjectMessageCreator` implementing `MessageCreator` looks more consistent and appropriate.